### DISCUSSION TITLE
Problema com frame.page.ts

### DISCUSSION THREAD

**Discussion by GabrielWerlich** [balta-io/7187#79] [Perguntas e Respostas] [answered]
Na aula "Masterpages" próximo ao minuto 07:40 você cria um framepage.ts  para renderizar o app-navbar e depois o router-outlet mas ele não permite eu compilar e dar sequencia nos próximos vídeos

*Detalhe estou utilizando a ultima versão do Angular, será que tem haver com isso?

Foto do código \/
![image](https://user-images.githubusercontent.com/40069738/158453010-51639489-4187-415c-8c05-8677b0d6f7c7.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @GabrielWerlich, como vai?
Por favor, utilize as mesmas versões do curso, pode ser que algo tenha mudado nas novas versões.
